common/server-datasource-ondexlocal/src/main/java/rres/knetminer/datasource/ondexlocal/OndexServiceProvider.java

```diff
@@ -163,8 +162,8 @@
      */
     private String sourceOrganization;
     
-    /*
-    * PubCount val
+    /**
+    * PubCount value
     */
     public static final String DEFAULT_EXPORTED_PUBS = "pubCount";
```

Comment:
something like OPT_DEFAULT_X_PUBS should be used. Moreover, "pubCount" as option name isn't clear, something like "defaultExportedPublications" would be better (options are usually more verbose than code names, since they address a less proficient audience).